docs/manual/src/proc_macro/index.md

```diff
@@ -228,6 +228,56 @@ fn do_http_request() -> Result<(), MyApiError> {
 }
 ```
 
+## The `#[uniffi::callback_interface]` attribute
+
+`#[uniffi::callback_interface]` can be used to export a [callback interface](../udl/callback_interfaces.html) definition
+This allows the foreign bindings to implement the interface and pass an instance to the Rust code.
+
+```rust
+#[uniffi::callback_interface]
+pub trait Person {
```

Comment:
Hm, why a separate macro? I think `#[uniffi::export]` or (`#[uniffi::export(callback_interface)]` if trait declarations can have a different meaning to UniFFI) would be more "natural", but of course that's very subjective.

Reply:
This was a very incisive comment and made me rethink a lot of how the code was organized.  I updated the code so not only does it share the same macro name, but the underlying uniffi macro code works more similarly.  In particular, callback interface traits and non-callback interface traits work very similarly which I think is a big win.